### ISSUE TITLE
[SCREENREADER]: Disability Calculator - actively manage focus when users press Calculate

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#431]
## Issue
Screen readers do not announce the calculated rate when users press Calculate. I propose we set focus when the Calculate button is clicked, so assistive tech users know the page has updated. Screenshot attached below.

## Acceptance Criteria
As a screenreader user, I would like to hear my calculated rate read aloud when I press the Calculate button.

## Environment
* MacOS && Win10
* Safari, Chrome, Firefox, IE11
* VoiceOver, NVDA, JAWS

## WCAG or Vendor Guidance (optional)
* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs
![Screen Shot 2019-06-27 at 4 43 58 PM](https://user-images.githubusercontent.com/934879/60303252-89587600-98fb-11e9-89d1-5bf9a8270c63.png)

**Comment by 1Copenut** [comment]
Closing per re-test on Staging: https://github.com/department-of-veterans-affairs/vets-website/pull/10345#pullrequestreview-256535970
